man/systemd.network.xml

```diff
@@ -1216,7 +1216,8 @@
         <varlistentry>
           <term><varname>Gateway=</varname></term>
           <listitem>
-            <para>As in the <literal>[Network]</literal> section.</para>
+            <para>Takes the gateway address or special value <literal>dhcp</literal>. If
+            <literal>dhcp</literal>, then the gateway address provided by DHCP is used.</para>
```

Comment:
hmm, maybe add "(or in the IPv6 case, provided by IPv6RA)"?

Reply:
Indeed. Will fix so.

Reply:
Updated.